cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/common/DefaultStorageProvider.java

```diff
@@ -23,12 +23,15 @@
 import io.cdap.cdap.common.conf.CConfiguration;
 import io.cdap.cdap.common.conf.Constants;
 import io.cdap.cdap.common.conf.SConfiguration;
+import io.cdap.cdap.data.runtime.StorageModule;
 import io.cdap.cdap.spi.data.StorageProvider;
 import io.cdap.cdap.spi.data.StructuredTableAdmin;
 import io.cdap.cdap.spi.data.nosql.NoSqlStorageProvider;
 import io.cdap.cdap.spi.data.sql.PostgreSqlStorageProvider;
 import io.cdap.cdap.spi.data.transaction.TransactionRunner;
 
+import javax.inject.Named;
```

Comment:
javax.inject.Named is standardized into Java as part of [JSR-330](http://javax-inject.github.io/javax-inject/) and [implemented by Guice](https://github.com/google/guice/wiki/JSR330). As it defines the standard for the Java language, it's probably preferable to use it over Guice-specific packages because Guice already implements these interfaces as well.